hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -22,6 +22,67 @@
       milestone: V1
       tests: ["i2c_sanity"]
     }
+    {
+      name: error
+      desc: '''
+            Program incorrect configuration into DUT register.
+
+            Stimulus:
+              - Program negative value into timing registers.
+              - Program incorrect format flags into fdata register.
+
+            Checking:
+              - Check involved interrupts/alerts.
+              - Got timeout in test (scoreboard might not get any transaction for DUT).
+            '''
+      milestone: V2
+      tests: ["i2c_error"]
+    }
+    {
+      name: reset_recovery
+      desc: '''
+            Pull reset at random times, make sure DUT recover/resets correctly and
+            there is no residual data left in the registers.
+
+            Stimulus:
+              - Randomly reset DUT.
+              - Restart send/receive transaction.
+
+            Checking:
+              - Check if registers are properly reset.
+            '''
+      milestone: V2
+      tests: ["i2c_reset_recovery"]
+    }
```

Comment:
you can add `{proj_root}/hw/dv/data/tests/stress_tests.hjson` in i2c_sim_cfg.hjson, then i2c_stress_all_with_rand_reset will be automatically added to the test list. Once your stress_all test is done, you can run `i2c_stress_all_with_rand_reset` which will test reset with stress_all test, including all the stimulus and checking you mentioned. see this [task](https://github.com/lowRISC/opentitan/blob/master/hw/dv/sv/cip_lib/cip_base_vseq.sv#L349)

perhaps change the name to `stress_all_with_reset` and tests to `i2c_stress_all_with_rand_reset`

Reply:
@rasmus-madsen & @senelson7 
FYI, above suggestion from @weicaiyang applies to AES & entropy_src as well.

Reply:
updated name and test according to the comment from @weicaiyang